pt-BR/active_record_validations.md

```diff
@@ -1234,12 +1234,12 @@ person.errors.details[:name]
 # => [{error: :invalid_characters, not_allowed: "!@#%*()_-+="}]
 ```
 
-All built in Rails validators populate the details hash with the corresponding
-validator type.
+Todos os validadores embutidos no *Rails* populam o *hash* de detalhes com o 
```

Comment:
```suggestion
Todos os validadores (*validators*)  embutidos no Rails populam o *hash* de detalhes com o 
```

Como Rails é nome próprio a gente não costuma colocar em itálico. Pode deixar normalmente mesmo :)